tools/fastdds/fastdds.bat

```diff
@@ -22,12 +22,13 @@ if not %ERRORLEVEL%==0 (
 )
 
 :: Python version in the form "Major.Minor"
-if %PYTHON_VERSION%=="" (
-      echo error retrieving python version. Please, make sure python is installed and accessible.
-      exit /B 65
-   )
+if not "%PYTHON_VERSION%" == "" (
+      :: Use launcher to profit from shebang hints on fastdds.py
+      :: Select the correct python version to source the appropriate paths
+      py -%PYTHON_VERSION% "%dir%\..\tools\fastdds\fastdds.py" %args%
```

Comment:
Non related with the PR, so it can be done in a follow-up PR.

Instead of the loop in lines 5-13, we can use `%*` here:
```suggestion
      py -%PYTHON_VERSION% "%dir%\..\tools\fastdds\fastdds.py" %*
```

Reply:
Will do in a follow-up PR